src/components/socketIoTest/MessageForm.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState, useRef } from 'react'
+import { socket } from '@/socket'
+
+export default function MessageForm({ room: roomId }: { room: string }) {
+	const [message, setMessage] = useState('')
+
+	const [isLoading, setIsLoading] = useState(false)
+	const messageRef = useRef<HTMLInputElement>(null)
+
+	function handleSend(event: React.FormEvent) {
+		event.preventDefault()
+		setIsLoading(true)
```

Comment:
bug: I believe you forgot to reset the `isLoading` state here ☺️

Reply:
Yes! Otherwise we can't submit a second message.

Reply:
Done

Reply:
New commit submitted.